todo_app/data/item.py

```diff
@@ -1,11 +1,11 @@
 from todo_app.data.itemstatus import ItemStatus
 
 class Item:
-    def __init__(self,id = None,status=None,title=None,cardid=None):
+    def __init__(self,id = None,status=None,title=None):
         self.id= id if id is not None else -1
         self.status= status if status is not None else ItemStatus.TODO
         self.title= title if title is not None else 'List saved todo items'
-        self.cardid= cardid
+        self.cardid= id
```

Comment:
You're just setting this to the same as the `id` field - is there any point having both? In particular, this never actually matches the `cardid` field that you set when creating a card (which is fine, but probably a hint we shouldn't bother setting that value!)

Reply:
Hi, Id and cardID are both used in the view. For legacy reasons i will leave both and assign them.